sources/launcher/Stride.Launcher/ViewModels/PackageVersionViewModel.cs

```diff
@@ -317,7 +317,7 @@ private async Task DeleteInternal(bool displayErrorMessage)
 
         private void UpdateStatusInternal()
         {
-            CanBeDownloaded = LocalPackage == null || (LocalPackage != null && ServerPackage != null && LocalPackage.Version < ServerPackage.Version);
+            CanBeDownloaded = (LocalPackage == null && ServerPackage != null) || (LocalPackage != null && ServerPackage != null && LocalPackage.Version < ServerPackage.Version);
```

Comment:
Somewhat ironic that the logic setting CanBeDownloaded never checked whether there was a server package to download or not.

I discovered this because there is no 4.1.x package for the VS extension.